src/types/Persons.ts

```diff
@@ -1,11 +1,11 @@
-export interface Person {
+export interface Persons {
```

Comment:
It's an interface for only one person, people are an array of persons - `type people = person[];` but it's not necessary to create an additional type